cves/CVE-2017-12615.yml

```diff
@@ -43,7 +43,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
```

Comment:
I could be wrong but I think there should be an indent before each of these lines while using the pipe character.

Reply:
Looks like I accidentally used spaces instead of tabs when writing this, thanks!